.github/workflows/nightly-it-example-generator.yml

```diff
@@ -89,6 +68,8 @@ jobs:
         run: ./mvnw versions:resolve-ranges -ntp -Dincludes='org.springframework:*,org.springframework.boot:*'
       - name: Prepare environments
         run: sh .github/workflows/resources/sql/nightly-generator-init.sh
+      - name: Build with Maven
```

Comment:
if you executing `mvn clean install` command, there should be a cache. it will speed up your dependency.